data/json/recipes/other/materials.json

```
@@ -1346,5 +1346,25 @@
       [ [ "forge", 60 ] ]
     ],
     "components": [ [ [ "glass_shard", 2 ] ], [ [ "chem_sulphur", 1 ] ], [ [ "scrap", 1 ] ], [ [ "charcoal", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "clockworks",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_PARTS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "skills_required": [ "mechanics", 2 ],
+    "time": "1 h 30 m",
+    "autolearn": [ [ "mechanics", 2 ] ],
+    "book_learn": [ [ "textbook_mechanics", 4 ], [ "manual_mechanics", 4 ], [ "textbook_fabrication", 4 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 1 },
+      { "id": "CUT_FINE", "level": 1 },
+      { "id": "DRILL", "level": 1 },
```

```suggestion
    "qualities": [
      { "id": "HAMMER_FINE", "level": 1 },
      { "id": "CUT_FINE", "level": 1 },
      { "id": "DRILL", "level": 2 },
```
Since it already mandates a full-on hacksaw and screwdriver set, might as well upgrade these tools a bit. Plus, not often a recipe actually gets to call hammer_fine instead of just max-tier hammer. :3